src/overlays/misc/ovl_kaleido_scope/z_kaleido_debug.c

```diff
@@ -53,35 +180,35 @@ void KaleidoScope_DrawDebugEditorText(Gfx** gfxP) {
 
     GfxPrint_SetPos(&printer, 4, 2);
```

Comment:
With Inventory Editor now being a thing, we can do for example `KaleidoScope_DrawDebugEditorText` -> `KaleidoScope_DrawInventoryEditor`

(I placed this review on a random line, ignore what I highlighted)